style/stylesheets/betterttv-dark.css

```diff
@@ -2156,3 +2156,32 @@ span.subscribe-text {
 .manager .meta .actions {
     border-color: #676767;
 }
+
+/* creative tag filter arrows */
+.creativetag-list-contain .filter-nav {
+    background: #333;
+}
+
+.creativetag-list-contain .filter-nav:before {
+    margin-top: -9px;
+    border: 5px solid #fff;
+    border-top-color: transparent;
+    border-bottom-color: transparent;
+    opacity: .5;
+}
+
+.filter-nav.goleft {
```

Comment:
That means nothing to me, since Twitch re-uses classes in multiple different CSS files and we have no way to verify it won't break now or in the future (hence the "may cause theme conflicts elsewhere").